src/Microsoft.Azure.WebJobs.Host/Bindings/SystemBindingData.cs

```diff
@@ -36,14 +36,14 @@ internal class SystemBindingData
         public string MethodName { get; set; }
 
         /// <summary>
-        /// Get the current UTC date. 
+        /// Get the current UTC date. This returns the current value each time it is called.
         /// </summary>
-        public DateTime UtcNow => DateTime.UtcNow;
+        public DateTime DateTime => DateTime.UtcNow;
```

Comment:
I'm trying to keep these consistent with the existing built-ins. That's what I have now in this PR - the same identifiers work in both paths. I think it's fine that the fact that the DateTime is UTC is just implicit - that's what we have now for {datetime}.